sources/packages/web/src/services/AuthService.ts

```diff
@@ -4,8 +4,58 @@ import { AppConfig, ClientIdType } from "../types/contracts/ConfigContract";
 import { AppConfigService } from "./AppConfigService";
 import HttpBaseClient from "./http/common/HttpBaseClient";
 import { UserService } from "./UserService";
+import { InstitutionService } from "./InstitutionService";
+import router from "../router";
+import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
+import { BCeIDDetailsDto } from "@/types";
 let keycloak: Keycloak.KeycloakInstance;
 
+async function navigateForInstitution(
+  authHeader: any,
+  bceIdAccountDetails: BCeIDDetailsDto,
+) {
```

Comment:
Please add the explicit return type Promise<boolean>.

Reply:
Promise has been explicitly defined.